website/blog/2025-05-20-podman-desktop-core.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Containers and Kubernetes development with Podman Desktop
+description: A step-by-step guide for your containers and Kubernetes development with Podman Desktop
+authors: [firewall]
+tags: [podman-desktop, podman, development, container, tools]
+hide_table_of_contents: false
+---
+
+In the world of modern software development, containers and Kubernetes are no longer optional; they are essential. That’s where Podman Desktop comes in, your ultimate tool for building, managing, and deploying containers and Kubernetes clusters with ease and confidence. In this blogpost we will walk through a typical development workflow, creating and building a container and then testing it in a local Kubernetes cluster. Let’s dive in.
+
+## Building Containerized Applications with Podman Desktop
+
+First, let's start by building an application in a container with Podman Desktop. We need our application code and a Containerfile. For a simple application, your Containerfile might look something like:
+
+```Dockerfile
+FROM nginxinc/nginx-unprivileged
```

Comment:
use `registry.access.redhat.com/ubi10/nginx-126`

Reply:
@benoitf 

When I try to run that one locally it's not working 
![image](https://github.com/user-attachments/assets/b4caed03-2164-45c0-81b1-e894e35b21c9)

For it to work you need to add a nginx config file and such (https://github.com/sclorg/nginx-container/blob/master/1.26/README.md#31-to-use-your-own-setup-create-a-dockerfile-with-this-content) (it doesn't come with a default one). I don't think it adds to the blogpost, just extra steps

Reply:
Whats wrong with the unpriveledged (non-root) nginx container?